layouts/partials/blocks/partners-page.html

```diff
@@ -110,6 +110,7 @@
                 {{ range $partnerCounties }}
                   <option value="{{ . }}">{{ . }} County</option>
                 {{ end }}
+                <option value="Regional/At-large">Regional/At-large</option>
```

Comment:
The big list of counties has "Regional/At-Large" in the middle. Maybe change the loop above to

```html
                {{ range $partnerCounties }}
                   {{ if ne . "Regional/At-larg" }}
                     <option value="{{ . }}">{{ . }} County</option>
                   {{ end }}
                 {{ end }}
```